local_storage.js

```diff
@@ -0,0 +1,12 @@
+// form object
+const contactForm = {
+  name: document.querySelector('#name').value,
+  email: document.querySelector('#email').value,
+  text: document.querySelector('#text').value,
+};
+
+// Store javascript objects in local storage
+window.localStorage.setItem('contactForm', JSON.stringify(contactForm));
```

Comment:
- [ ] The values passed into the input fields should not be erased after refreshing the page. Currently this is the case, it would be better to parse it back to  an object after retrieving it from localstorage using the following format:
 
``savedFormData = JSON.parse(savedFormData);`

- [ ] Also upon loading when the user loads the page, the code should check if there's any saved data in the local storage and if so the input fields should be prefilled with the data. Refer to the code snippet below for reference.
```
window.onload = () => {
  let savedFormData = localStorage.getItem('contactFormInput');
  savedFormData = JSON.parse(savedFormData);
  // Check if the form data object is found on localStorage
  if (savedFormData) {
  // populate inputs values if data was found
  // ex: nameInput.value = savedFormData.name
  }
};
```